documentation/Forms/FormViewer.md

```diff
@@ -0,0 +1,577 @@
+    - [5.5.3. [Optional] Property `GC Forms Schema - Alpha > form > brand > urlFr`](#form_brand_urlFr)
+    - [5.5.4. [Optional] Property `GC Forms Schema - Alpha > form > brand > logoEn`](#form_brand_logoEn)
+    - [5.5.5. [Optional] Property `GC Forms Schema - Alpha > form > brand > logoFr`](#form_brand_logoFr)
+    - [5.5.6. [Optional] Property `GC Forms Schema - Alpha > form > brand > logoTitleEn`](#form_brand_logoTitleEn)
+    - [5.5.7. [Optional] Property `GC Forms Schema - Alpha > form > brand > logoTitleFr`](#form_brand_logoTitleFr)
+    - [5.5.8. [Optional] Property `GC Forms Schema - Alpha > form > brand > disableGCBranding`](#form_brand_disableGCBranding)
+  - [5.6. [Optional] Property `GC Forms Schema - Alpha > form > emailSubjectEn`](#form_emailSubjectEn)
+  - [5.7. [Optional] Property `GC Forms Schema - Alpha > form > emailSubjectFr`](#form_emailSubjectFr)
+  - [5.8. [Optional] Property `GC Forms Schema - Alpha > form > endPage`](#form_endPage)
+    - [5.8.1. [Optional] Property `GC Forms Schema - Alpha > form > endPage > descriptionEn`](#form_endPage_descriptionEn)
+    - [5.8.2. [Optional] Property `GC Forms Schema - Alpha > form > endPage > descriptionFr`](#form_endPage_descriptionFr)
+    - [5.8.3. [Optional] Property `GC Forms Schema - Alpha > form > endPage > referrerUrlEn`](#form_endPage_referrerUrlEn)
+    - [5.8.4. [Optional] Property `GC Forms Schema - Alpha > form > endPage > referrerUrlFr`](#form_endPage_referrerUrlFr)
+- [6. [Required] Property `GC Forms Schema - Alpha > submission`](#submission)
+  - [6.1. [Optional] Property `GC Forms Schema - Alpha > submission > email`](#submission_email)
+  - [6.2. [Optional] Property `GC Forms Schema - Alpha > submission > vault`](#submission_vault)
+- [7. [Optional] Property `GC Forms Schema - Alpha > reCaptchaID`](#reCaptchaID)
+
+**Title:** GC Forms Schema - Alpha
+
+| Type                      | `object`                                                                  |
+| ------------------------- | ------------------------------------------------------------------------- |
+| **Additional properties** | [[Any type: allowed]](# "Additional Properties of any type are allowed.") |
+| **Default**               | `{}`                                                                      |
+
+**Description:** A schema to define a Forms configuration.
+
+| Property                                         | Pattern | Type             | Deprecated | Definition | Title/Description                                                                    |
+| ------------------------------------------------ | ------- | ---------------- | ---------- | ---------- | ------------------------------------------------------------------------------------ |
+| - [securityAttribute](#securityAttribute )       | No      | enum (of string) | No         | -          | It shows an administrator what proceduce should be taken when accessing a form's ... |
```

Comment:
Finding this a little difficult to read as its cut off

Reply:
It's only cut off in the summary of the attributes. It's not cut off in the section for the attribute itself.